pkg/webhook/internal/certwatcher/certwatcher.go

```diff
@@ -0,0 +1,162 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package certwatcher
+
+import (
+	"crypto/tls"
+	"sync"
+
+	"gopkg.in/fsnotify.v1"
```

Comment:
I'd suggest using `https://github.com/fsnotify/fsnotify` directly. CR is now using go modules, we don't need to use `gopkg.in`.

Reply:
I used the `gopkg.in` URL because it was already vendored this way as an indirect dependency. I see that the `Gopkg.*` files still exist, and that `dep check` is run as part of the test suite. `Gopkg.toml` references `gopkg.in/fsnotify.v1`.